composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "minimum-stability": "stable",
     "require": {
-        "symfony/framework-bundle": "~2.1",
+        "symfony/framework-bundle": "~2.1|~3.0",
         "nikic/php-parser": "~1.4.1",
         "symfony/console": "*"
```

Comment:
I'd agree with these suggestions - they just aren't in the scope of this specific PR, which is to add support for Symfony 3.